### ISSUE TITLE
~/.homebridge/config.json

### ISSUE THREAD

**Issue by richierockskool** [homebridge/homebridge#3357]
### Current Situation

No plugin was found for the platform "HomebridgeInkbirdWifiGateway" in your config.json. Please make sure the corresponding plugin is installed correctly.

### Logs

```markdown
https://github.com/richierockskool/homebridge-inkbird-wifi-gateway

All files and modules debugged, Homebridge installs, but can not find the plug-in for the  Platform? Please assist. Last piece of the puzzle.
```

### Configuration

```json
{"displayName":"Homebridge Inkbird Wifi Gateway",
"name":"homebridge-inkbird-wifi-gateway",
"author":"R.Jagger",
"version":"1.0.6",
"description":"Control Inkbird BT devices through the WiFi Gateway App",
"license":"Apache-2.0",
"type":"module",
"repository":
{"url":"https://github.com/richierockskool/homebridge-inkbird-wifi-gateway.git"},
"bugs":{"url":"https://github.com/richierockskool/homebridge-inkbird-wifi-gateway/issues"},"funding":[{"type":"github","url":"https://github.com/sponsors/richierockskool"},
{"type":"paypal",
"url":"https://www.paypal.com/paypalme/RichJagger"}],
"engines":{"node":">=16.17.0",
"homebridge":">=1.5"},
"main":"src/index.js",
"scripts":{"lint":"eslint src/**.ts --max-warnings=0",
"watch":"npm run build && npm link && nodemon",
"build":"rimraf ./dist && tsc",
"prepublishOnly":"npm run lint && npm run build"},
"keywords":
["homebridge-plugin",
"bluetooth",
"temperature",
"sensor",
"inkbird",
"homebridge"],
"devDependencies":
{"@types/node":"^16.10.9",
"@typescript-eslint/eslint-plugin":"^5.0.0",
"@typescript-eslint/parser":"^5.0.0",
"eslint":"^8.0.1",
"homebridge":"^1.3.9",
"nodemon":"^1.3.3",
"rimraf":"^3.0.2",
"ts-node":"^10.3.0",
"typescript":"^4.4.4"},
"dependencies":{"@abandonware/noble":"^1.9.2-21",
"chokidar":"^3.5.3",
"homebridge-inkbird-wifi-gateway":"github:richierockskool/homebridge-inkbird-wifi-gateway",
"latest-version":"^7.0.0",
"module-name":"^0.0.1-security",
"my-package":"^0.0.0"}
}
```

### Environment

- OS:
- Software:
- Node:
- npm:

### Process Supervisor

hb-service

**Comment by NorthernMan54** [comment]
The file your shared is not a config.json, are you using the Homebridge Image and the homebridge UI to configure your environment ?

**Comment by richierockskool** [comment]
{
  "pluginAlias": "HomebridgeInkbirdWifiGateway",
  "pluginType": "platform",
  "singular": true,
   "schema": {
    "type": "object",
    "properties": {
      "name": {
        "title": "Name of the sensor",
        "type": "string",
        "required": true,
        "default": "Homebridge Inkbird Wifi Gateway",
        "description":"HomebridgeInkbirdWifiGateway"
        
      },
      "model": {
            "type": "string",
            "enum": [ "IBS-M1", "not in list - try it anyway" ],
            "title": "Model number (i.e. IBS-M1, leave empty if not sure)",
            "required": true
    },
      "mac_address": {
            "type": "string",
            "title": "Bluetooth MAC address for temperature sensor - WiFi gateway (xx:xx:xx:xx:xx:xx). If empty, then first sensor to be found will be used - see log for MAC found.",
            "required": false
  },
      "update_interval": {
            "type": "number",
            "title": "Update interval in seconds",
            "required": false,
            "minimum" : 15
},
      "storage": {
            "type": "string",
            "enum": ["filesystem", "googleDrive"],
            "title": "Storage of history values (filesystem or googleDrive)",
            "required": false
        },
         "loglevel": {
            "type": "number",
            "title": "The log level at start - smaller number less messages (0 = Fatal, 1 = Error, 2 = Warning, 3 = Info (default), 4 = Debug)",
            "required": false,
            "minimum" : 0,
            "maximum" : 4
         }
    }
  }
}

**Comment by richierockskool** [comment]
and here's the last error...
Error: require() of ES Module /app/available_modules/1682646873000/homebridge-inkbird-wifi-gateway/index.js from /app/index.js not supported.
Instead change the require of /app/available_modules/1682646873000/homebridge-inkbird-wifi-gateway/index.js in /app/index.js to a dynamic import() which is available in all CommonJS modules.

**Comment by NorthernMan54** [comment]
Are you the developer of this plugin ?

**Comment by richierockskool** [comment]
Yes, my first one. I made many adjustments recently and now Homebridge can find the plugin, and load this plugin.
but do to some coding errors that I do not understand yet,  it crashes the whole Homebridge UI when I install (I have to Disable - Homebridge Inkbird Wifi Gateway to get it work again). So, I am just trying to get another pair of eyes on it to delete what I do not need and fix the remaining errors.

Thank you,

Rich

**Comment by richierockskool** [comment]
More errors are gone now since I have correct some..now get the failed to load config schema json
